src/bootstrap-table.js

```diff
@@ -919,7 +919,7 @@ class BootstrapTable {
 
       html.push(`<div class="${this.constants.classes.pull}-${o.paginationHAlign} pagination">`,
         Utils.sprintf(this.constants.html.pagination[0], Utils.sprintf(' pagination-%s', o.iconSize)),
-        Utils.sprintf(this.constants.html.paginationItem, ' page-pre', o.paginationPreText))
+        Utils.sprintf(this.constants.html.paginationItem, ' page-pre', o.formatSRPaginationPreText(false, 'zzzz'), o.paginationPreText))
```

Comment:
What is the purpose of `false, 'zzzz'` parameters?

Reply:
Im not sure right now (i dont have the IDE open), i will check it later.

Reply:
i think it was an rebase issue...
i removed it!